backend/src/bundles/users/user.service.ts

```diff
@@ -63,8 +63,8 @@ class UserService implements IService {
         return updatedUser.toObject();
     }
 
-    public delete(): ReturnType<IService['delete']> {
-        return Promise.resolve(true);
+    public async deleteUser(id: string): Promise<UserEntity | undefined> {
```

Comment:
I not sure that we should return `UserEntity` on delete

Reply:
fixed